src/cross_toml.rs

```diff
@@ -0,0 +1,222 @@
+//! Implements the parsing of `Cross.toml`
+//!
+//! `Cross.toml` can contain the following elements:
+//!
+//! # `build`
+//! The `build` key allows you to set global variables, e.g.:
+//!
+//! ```toml
+//! [build]
+//! xargo = true
+//! ```
+//!
+//! # `build.env`
+//! With the `build.env` key you can globally set volumes that should be mounted
+//! in the Docker container or environment variables that should be passed through.
+//! For example:
+//!
+//! ```toml
+//! [build.env]
+//! volumes = ["vol1", "vol2"]
```

Comment:
this should be upper case, it's also environment variables

```suggestion
//! volumes = ["VOL1_ARG", "VOL1_ARG"]
```